sway/config.c

```diff
@@ -56,6 +56,12 @@ static void free_mode(struct sway_mode *mode) {
 		}
 		list_free(mode->mouse_bindings);
 	}
+	if (mode->switch_bindings) {
+		for (int i = 0; i < mode->switch_bindings->length; i++) {
+			free_switch_binding(mode->mouse_bindings->items[i]);
```

Comment:
s/mouse/switch/

Reply:
That explains that mysterious crash on config reload... What happens when you're rushing before the kids are up from their naps.